specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlPool.json

```diff
@@ -4211,6 +4211,180 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/sqlPools/{sqlPoolName}/maintenanceWindows/current": {
+      "get": {
+        "tags": [
+          "SqlPools, MaintenanceWindows"
+        ],
+        "description": "Gets maintenance windows settings for a Sql Pool.",
+        "operationId": "MaintenanceWindows_Get",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./workspace.json#/parameters/workspaceNameParameter"
+          },
+          {
+            "type": "string",
+            "in": "path",
+            "name": "sqlPoolName",
+            "description": "SQL pool name",
+            "required": true
+          },
+          {
+            "name": "maintenanceWindowName",
+            "in": "query",
+            "description": "Maintenance window name.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved maintenance windows settings.",
+            "schema": {
+              "$ref": "#/definitions/MaintenanceWindows"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***"
+          }
+        },
+        "x-ms-examples": {
+          "Gets maintenance window settings for a selected Sql Pool.": {
+            "$ref": "./examples/GetMaintenanceWindows.json"
+          }
+        }
+      },
+      "put": {
```

Comment:
no delete? so a customer can create a maintenance window, but not delete it? how do they undo the put?